crates/aptos-rest-client/src/lib.rs

```diff
@@ -608,7 +608,7 @@ impl Client {
             if elapsed.as_secs() > 30 {
                 sample!(
                     SampleRate::Duration(Duration::from_secs(30)),
-                    warn!(
+                    debug!(
```

Comment:
This one I would leave.  It prints after waiting for more than 30s, so person doesn't think it is stuck?

Reply:
(it prints after 30s, and then every 30s later, while waiting)

Reply:
I think in general since this is the backbone of the future SDK, these kinds of messages should be opt in for debugging purposes.

I'm putting it down to debug right now, because we need to do a revamp on how we handle logging in general in the client, sdk, and CLI as we approach a renovated Rust SDK

Reply:
You think people will not have a problem for CLI hanging for 1-3 minutes,  without any signal it is still doing something?

Anyways,  your call.